### DISCUSSION TITLE
PGID/PGID ?

### DISCUSSION THREAD

**Discussion by lowrisk75** [alexbelgium/hassio-addons#566] [Q&A] [answered]
I'm migrating sonarr/radarr from a Synology NAS to a VM for Home Assistant, i'm just confused with the PGID/PGID value.
I'm running home assistant + supervisor on a debian image (within a UTM VM).

where do I found the correct PGID/PGID ? 
also I'm wonder how are the container isolated from HA ? (they all in the same IoT VLAN in my setup anyway)

**Comment by alexbelgium** [comment, marked as answer]
Hi, here is a guide : https://docs.linuxserver.io/general/understanding-puid-and-pgid
